File: 0283-move-zeroes/0283-move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        arr = []
        for i in range(len(nums)):
            if nums[i] != 0:
                arr.append(nums[i])
        
        for j in range(len(arr),len(nums)):
            arr.append(0)
        
        nums[:] = arr

        

File: 0283-move-zeroes/0283-move-zeroes.cpp
class Solution {
public:
    void moveZeroes(vector<int>& nums) {
        vector<int> arr1;
        for(int i=0; i<nums.size(); i++){
            if (nums[i] != 0){
                arr1.push_back(nums[i]);
            }
        }
        for(int j=arr1.size(); j<nums.size(); j++){
            arr1.push_back(0);
        }
        nums = arr1;
    }
};